eslint_plugin/eslint_plugin_report.js

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+const { getAnnotations } = require('./eslint_plugin_annotations')
+const { countIssueLevels } = require('../annotations_levels')
+
+/**
+ * Process Eslint plugin output (generate annotations, count issue levels)
+ * @param {*} results Eslint plugin json output
+ */
+module.exports = (results) => {
+  // The structure of the ESlint JSON output is that
+  // the issues found by ESLint are grouped by files.
+  // In the ESlint output there are files without any problems
+  // and that's why I have to filter them.
```

Comment:
It sounds strange to use "I" in code comments because nobody knows who that is.  Avoid using pronouns and just describe the code.

Reply:
I will change that.